lib/remote_record/reference.rb

```diff
@@ -47,6 +47,13 @@ def remote_where(params, &authz_proc)
         find_or_initialize_all(remote_record_class.where(params, &authz_proc))
       end
 
+      def remote_find_by(params, &authz_proc)
+        return remote_where(params, &authz_proc).first unless remote_record_class.respond_to?(:find_by)
+
+        resource = remote_record_class.find_by(params, &authz_proc)
+        new(remote_resource_id: resource['id'], initial_attrs: resource)
```

Comment:
this is assuming the name of the field is `id`

Reply:
Raised #36